pkg/chrootuser/user.go

```diff
@@ -70,6 +70,20 @@ func GetUser(rootdir, userspec string) (uint32, uint32, error) {
 	return 0, 0, err
 }
 
+// GetGroup returns the gid by looking it up in the /etc/passwd file
+// groupspec format [ group | gid ]
+func GetGroup(rootdir, groupspec string) (uint32, error) {
+	var gid64 uint64
+	var gerr error = user.UnknownGroupError("error looking up group")
+	gid64, gerr = strconv.ParseUint(groupspec, 10, 32)
+	if gerr != nil {
+		// The group couldn't be parsed as a number, so look up
+		// the group's GID.
+		gid64, gerr = lookupGroupInContainer(rootdir, groupspec)
+	}
+	return uint32(gid64), errors.Wrapf(gerr, "error detrmining run gid %q", groupspec)
```

Comment:
Does errors.Wrapf(nil, "FOOBAR")
Return the error == nil or an error with FOOBAR?

Reply:
It returns error == nil.

Reply:
Yup, that was a mistake. Fixed now. Thanks!

Reply:
But now gerr isn't declared; is Go not strictly typed?

Reply:
Sorry if ^^ is a stupid question. /me resolves to delve into Go one of these weekends.

Reply:
`gid64, gerr := strconv.ParseUint(groupspec, 10, 32)`
It is declared now, the : in front of the = means it is a declaration in Go. It automatically picks up the type depending on what the right side of the equation returns.